docs/api/touch-bar.md

```diff
@@ -6,14 +6,25 @@ Process: [Main](../tutorial/quick-start.md#main-process)
 
 ### `new TouchBar(items)` _Experimental_
 
-* `items` ([TouchBarButton](touch-bar-button.md) | [TouchBarColorPicker](touch-bar-color-picker.md) | [TouchBarGroup](touch-bar-group.md) | [TouchBarLabel](touch-bar-label.md) | [TouchBarPopover](touch-bar-popover.md) | [TouchBarSlider](touch-bar-slider.md) | [TouchBarSpacer](touch-bar-spacer.md))[]
+* `items` ([TouchBarButton](touch-bar-button.md) | [TouchBarColorPicker](touch-bar-color-picker.md) | [TouchBarGroup](touch-bar-group.md) | [TouchBarLabel](touch-bar-label.md) | [TouchBarPopover](touch-bar-popover.md) | [TouchBarScrubber](touch-bar-scrubber.md) | [TouchBarSegmentedControl](touch-bar-segmented-control.md) | [TouchBarSlider](touch-bar-slider.md) | [TouchBarSpacer](touch-bar-spacer.md))[]
 
 Creates a new touch bar with the specified items. Use
 `BrowserWindow.setTouchBar` to add the `TouchBar` to a window.
 
 **Note:** The TouchBar API is currently experimental and may change or be
 removed in future Electron releases.
 
+### Instance Methods
+
+The following methods are available on instances of `TouchBar`:
+
+#### `touchBar.setEscapeItem([touchBarItem])`
```

Comment:
Hmm, thinking about this a bit more, should this be an `escapeItem` property instead? The items use properties instead of setters, what do you think @MarshallOfSound?